src/views/test/GeneratedCustomers.tsx

```diff
@@ -1,12 +1,59 @@
-import data from '../customer/CustomerListView/data'
 import React, { useState } from 'react';
 import GeneratedTable from './GeneratedCustomersTable'
+import { useFilterCustomerGridDataQuery } from 'src/generated/graphql'
+// import { Customer_Order_By } from 'src/generated/graphql'
 
 const GeneratedCustomers = () => {
-  const [customers] = useState(data);
+  const [sort, setSortCustomers] = React.useState(
+    {
+      name: 'asc'
+    }
+  );
+  const [limit, setLimit] = useState<number>(2);
```

Comment:
Let's just have states: `page` and `pageSize` instead of three (`page`, `limit`, `offset`)

Reply:
created only 2 states **page** and **pageSize**.  Calculation of offset is only  **const offset = page * pageSize**